pkg/distributor/ha_tracker.go

```diff
@@ -150,6 +150,12 @@ func (c *haTracker) loop(ctx context.Context) {
 		c.electedLock.Lock()
 		defer c.electedLock.Unlock()
 		chunks := strings.SplitN(key, "/", 2)
+
+		// Only handle valid keys.
```

Comment:
Can we leave a comment here or somewhere nearby like `The prefix has already been stripped, so a valid key would look like cluster/replica, and a key without a / such as ring would be invalid.`